docs/sections/user_guide/installation.rst

```diff
@@ -1,5 +1,67 @@
+.. include:: ../links.rst
+
 ************
 Installation
 ************
 
-TBD
+
+-  If you are a Developer, skip the User Installation and continue on to the Developer Setup located in the Contributer Guide.
+
+The recommended installation mechanism uses the Python package and virtual-environment manager `conda`_. Specifically, these instructions detail the use of the minimal `Miniforge`_ variant of `Miniconda`_, built to use, by default, packages from the `conda-forge`_ project. 
+
+Users of the original Miniconda (or the `Anaconda`_ distribution) may need to add the flags ``-c conda-forge --override-channels`` to ``conda build``, ``conda create``, and ``conda install`` commands to specify the use of conda-forge packages.
+
+
+Using a fresh Miniforge installation
+====================================
+
+This recipe uses the ``aarch64`` (64-bit ARM) Miniforge for Linux, and installs into ``$HOME/conda``. Adjust as necessary for your target system.
+
+1. Download, install, and activate the latest `Miniforge3`_ for your system. If an existing conda (Miniforge, Miniconda, Anaconda, etc.) installation is available and writable, you may activate that and skip this step and continue on to the next.
+
+
+   .. code:: sh
+
+     wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh
+     bash Miniforge3-Linux-aarch64.sh -bfp ~/conda
+     rm Miniforge3-Linux-aarch64.sh
+     source ~/conda/etc/profile.d/conda.sh
+     conda activate
```

Comment:
If I'm not mistaken, the text of bullet 1 and then this code block are identical to what's in the Contributor Guide. Could we factor that out to a shared `.rst` file and include it in both places so that we only have to maintain this fairly complex set of instructions once?